R/tribble.R

```diff
@@ -1,8 +1,9 @@
 #' Row-wise tibble creation
 #'
-#' Create \code{\link{tibble}}s laying out the data in rows, rather than
-#' in columns. This is useful for small tables of data where readability is
-#' important.  Please see \link{tibble-package} for a general introduction.
+#' Create \code{\link{tibble}}s by inputing the data observations row-by-row,
+#' rather than column-by-column.  This is useful for small tables of data
```

Comment:
How about keeping the first sentence as-is, and adding a second sentence that explains  slightly differently? We could also expand the examples, so that the intention of tribble() is clearer.